src/puppetlabs/puppetdb/reports.clj

```diff
@@ -65,19 +80,19 @@
 (def report-v3-wireformat-schema
   (dissoc report-v4-wireformat-schema :status))
 
-(pls/defn-validated sanitize-events :- [resource-event-wireformat-schema]
+(pls/defn-validated sanitize-v5-resource-events :- [resource-event-v5-wireformat-schema]
   "This function takes an array of events and santizes them, ensuring only
    valid keys are returned."
-  [events :- (s/pred coll? 'coll?)]
-  (for [event events]
-    (pls/strip-unknown-keys resource-event-wireformat-schema event)))
+  [resource-events :- (s/pred coll? 'coll?)]
+  (for [resource-event resource-events]
+    (pls/strip-unknown-keys resource-event-v5-wireformat-schema resource-event)))
```

Comment:
You missed a zero-point chance!
(map (partial pls/strip-unknown-keys resource-event-v5-wireformat-schema) resource-events)